docs/JupyterNotebook.md

```diff
@@ -1,8 +1,80 @@
 # Talk2PowerSystem Jupyter Notebook
 
-##  Run the notebook
+## Run the chatbot on your developer machine using a Jupyter notebook
 
 ```bash
+conda create --name Talk2PowerSystemLLM --file conda-linux-64.lock
 conda activate Talk2PowerSystemLLM
+poetry install
 jupyter notebook
 ```
+
+The notebook will prompt you for the GraphDB and Azure OpenAI credentials.
+  - Graphdb password is shared in Graphwise Keeper, entry `DATA/CIM/CIM GraphDB`
+  - Azure OpenAI credentials are shared in Graphwise Keeper, entry `Statnett/Graphwise Azure Statnett Deployment GPT4.1`
+
+## Run the chatbot on the RNDP environment using a Jupyter notebook
+
+- Open https://rndp.statnett.no/ from your browser.
+- Login with you Statnett Microsoft Account.
+- A screen like this should appear:
+![server_options_screen.png](images/server_options_screen.png)
+- For size pickup `Small` and click `Start`
+- It takes a while to start the server
+ ![server_starting.png](images/server_starting.png)
+- After that you should see a screen like this:
+![landing.png](images/landing.png)
+- From the left menu select git:
+![git.png](images/git.png)
+- Click the blue button `Clone a Repository`, enter https://github.com/statnett/Talk2PowerSystem_LLM for URI and click `Clone`:
+![clone.png](images/clone.png)
+- A folder named `Talk2PowerSystem_LLM` should appear under your home directory `/home/{your-user}`:
+![repository.png](images/repository.png)
+- From the `Launcher` tab section `Other` open a terminal
+- A screen like this should appear:
+![terminal.png](images/terminal.png)
+- Check your working directory:
+```commandline
+pwd
+```
+It should be your home directory.
+- Setup the environment:
+  - Execute:
+    ```commandline
+    cd Talk2PowerSystem_LLM
+    python3 -m venv .venv
+    source .venv/bin/activate
+    ```
+  - Change the priority of the `statnett` mirroring repository from `supplemental` to `primary` using `nano` text editor:
```

Comment:
Unclear what needs to change here. The toml file doesn't have "priority" or "supplemental" anywhere and trying to add priority="primary" in the [tool.poetry] section causes the following error:

The Poetry configuration is invalid:
  - Additional properties are not allowed ('priority' was unexpected)

Reply:
Something goes wrong with the dependencies without this set (unsurprisingly) and I get "All attempts to connect to pypi.org failed." since RNDP can't connect to pypi.org

Looking through the poetry documentation though, sections with tool.poetry.source would by default be primary

> If priority is undefined, the source is considered a primary source, which disables the implicit PyPI source and takes precedence over supplemental sources.

Issue being we have no source section in the toml file and that's the only place where a priority can be specified as far as I can tell.

Reply:
The instructions assume that the changes form this branch are merged to master. Until then we should checkout the branch statnett-66 and follow the instructions from there